infrastructure/server/oidc_clients_fom.tf

```diff
@@ -7,6 +7,9 @@ resource "aws_cognito_user_pool_client" "dev_fom_oidc_client" {
     "https://oidcdebuggersecure-3d5c3f-dev.apps.silver.devops.gov.bc.ca/",
     "http://localhost:4200/admin"
   ]
+  logout_urls                                   = [
+    "${var.frontend_logout_chain_url}http://localhost:4200/admin/not-authorized?loggedout=true"
+  ]
```

Comment:
I assume all terraform files can read the variable "var.frontend_logout_chain_url"?

Reply:
Yes, they should be able to. The Terraform CI jobs would fail if they can't find a variable.